spec/03-plugins/08-datadog/02-schema_spec.lua

```diff
@@ -8,6 +8,12 @@ describe("Plugin: datadog (schema)", function()
     assert.is_nil(err)
     assert.is_truthy(ok)
   end)
+  it("accepts nil host and port with use_env", function()
+    -- Defaults are set for host and port, as we're not really testing for undefined values
```

Comment:
you can force them to `null` probably by setting them to `ngx.null` instead of `nil`